eng/pipelines/dailytests.yml

```diff
@@ -0,0 +1,31 @@
+resources:
+  pipelines:
+  - pipeline: ComponentBuildUnderTest
+    source: NuGet.Client-Official
+    trigger:
+        branches:
+          - dev-*
```

Comment:
```suggestion
          - dev
```

Reply:
So this would effectively trigger with the weekly builds?

Reply:
No, it would trigger every time a new commit is added to the `dev` branch. The way that this PR currently has it, `dev-*`, it would not trigger when the `dev` branch changes, but will only tigger on` dev-*` branch changes. Considering most of us push `dev-*` changes multiple times every day, this would be an enormous waste of resources, and then it wouldn't even test the product code that we ship (the dev/main branch).